openlife/client/Client.hx

```diff
@@ -88,8 +89,15 @@ class Client
 		{
 			if(e.message != "Blocked")
 			{
-                trace('e: ${e.details()}');
-                close();
+                if(e.details().indexOf('Eof')>-1){
```

Comment:
Not entirely needed - just to stop nasty exception messages from being thrown. Should work without.